docker-compose.yml

```
@@ -39,7 +39,7 @@ services:
       - kuzzle_server__protocols__mqtt__developmentMode=false
       - kuzzle_limits__loginsPerSecond=50
       - NODE_ENV=${NODE_ENV:-development}
-      - DEBUG=kuzzle:*,-kuzzle:entry-point:protocols:websocket
+      - DEBUG=${DEBUG:-kuzzle:*,-kuzzle:entry-point:protocols:websocket}
```

This does not work, try this:
```suggestion
      - DEBUG=$DEBUG,-kuzzle:*,-kuzzle:entry-point:protocols:websocket
```

---

I used this syntax because I want to use the value of `DEBUG` is set and otherwise I want `kuzzle:*,-kuzzle:entry-point:protocols:websocket`